source/nodejs/adaptivecards/src/utils.ts

```diff
@@ -44,109 +40,44 @@ export function getBoolValue(value: any, defaultValue: boolean): boolean {
     return defaultValue;
 }
 
-export function getEnumValue(targetEnum: { [s: number]: string }, name: string, defaultValue: number): number {
-    if (isNullOrEmpty(name)) {
-        return defaultValue;
-    }
-
-    for (var key in targetEnum) {
-        let isValueProperty = parseInt(key, 10) >= 0
+export function getEnumValueByName(enumType: { [s: number]: string }, name: string) : number | undefined {
+    for (let key in enumType) {
+        let keyAsNumber = parseInt(key, 10);
```

Comment:
>let [](start = 8, length = 3)

could be `const` instead of `let`

Reply:
Sure, but that's a nit and if I do it here I would have to do it everywhere. Will keep the lets for the time being.